go/common/log_events.go

```diff
@@ -5,20 +5,13 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/eth/filters"
 	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/ten-protocol/go-ten/go/common/viewingkey"
 )
 
 // LogSubscription is an authenticated subscription to logs.
 type LogSubscription struct {
-	// The account the events relate to.
-	Account *common.Address
```

Comment:
no need to send this, because it can be retrieved from the  signature over the VK.
Even if sent, It has to be checked to match the retrieved key, and creates a security risk